### PR TITLE
Fix the gRPC user-guide URL and name

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/3882

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
 Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool.
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [the gRPC user guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline README.md:38]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: ballerina/Module.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
  Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool. 
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [the gRPC user guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline ballerina/Module.md:31]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: ballerina/Package.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
 Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool.
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [the gRPC user guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline ballerina/Package.md:31]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: README.md (hunk 1 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/60af9102b24b4960998e0b9ca322e6a7602ae741

File: ballerina/Module.md (hunk 2 above)
Comment (BuddhiWathsala) [reply ballerina/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/60af9102b24b4960998e0b9ca322e6a7602ae741

File: README.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
 Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool.
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline README.md:38]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: ballerina/Module.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
  Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool. 
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline ballerina/Module.md:31]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: ballerina/Package.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
 Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool.
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```
Comment (praneesha) [inline ballerina/Package.md:31]:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

File: README.md (hunk 4 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/eaacf072339c9d9df83999e0d82f1d6dd8b8acce

**Review by praneesha — APPROVED** [review]

File: ballerina/Module.md (hunk 5 above)
Comment (BuddhiWathsala) [reply ballerina/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/eaacf072339c9d9df83999e0d82f1d6dd8b8acce

File: ballerina/Package.md (hunk 6 above)
Comment (BuddhiWathsala) [reply ballerina/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/eaacf072339c9d9df83999e0d82f1d6dd8b8acce

**Pull request merged**